src/js/backend.ts

```diff
@@ -115,6 +118,38 @@ export class ReactNativeBackend extends BaseBackend<BrowserOptions> {
     }
 
     this._options.onReady?.({ didCallNativeInit });
+
+    if (didCallNativeInit) {
+      const {
+        nativeStartTime,
+        nativeStartedTime,
+      } = await NATIVE.getNativeStartupTimestamps();
+
+      const nativeStartTimeSeconds = nativeStartTime / 1000;
+      const nativeStartedTimeSeconds = nativeStartedTime / 1000;
+
+      const t = getCurrentHub().startTransaction({
```

Comment:
these are metrics, not a transaction, so ideally we'd not start transaction and spans, but rather sent along with the App's transactions